pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1213,24 +1267,33 @@ func TestReconcileInvalidTaskRuns(t *testing.T) {
 			defer cancel()
 			c := testAssets.Controller
 			clients := testAssets.Clients
-			err := c.Reconciler.Reconcile(context.Background(), getRunName(tc.taskRun))
+			reconciler := c.Reconciler.(*Reconciler)
+			fr := reconciler.Recorder.(*record.FakeRecorder)
+			err := reconciler.Reconcile(context.Background(), getRunName(tc.taskRun))
+
 			// Events are sent in a goroutine, let's sleep a bit to make sure they're
 			// captured by the fake client-go action list
-			time.Sleep(100 * time.Millisecond)
+			time.Sleep(2000 * time.Millisecond)
+
 			// When a TaskRun is invalid and can't run, we don't want to return an error because
 			// an error will tell the Reconciler to keep trying to reconcile; instead we want to stop
 			// and forget about the Run.
 			if err != nil {
 				t.Errorf("Did not expect to see error when reconciling invalid TaskRun but saw %q", err)
 			}
+
+			// Check actions and events
 			actions := clients.Kube.Actions()
-			if len(actions) != 2 ||
-				actions[0].GetVerb() != "list" ||
-				actions[0].GetResource().Resource != "namespaces" ||
-				actions[1].GetVerb() != "create" ||
-				actions[1].GetResource().Resource != "events" {
-				t.Errorf("expected two actions (list namespaces + event) created by the reconciler, got %+v", actions)
+			if len(actions) != 1 ||
+				actions[0].Matches("namespaces", "list") {
```

Comment:
one line maybe?

THE MOST MINOR COMMENT OF ALL TIME

Reply:
😅 and I even thought about it, not sure why I did not do it?